lsp-mode.el

```diff
@@ -6426,31 +6431,33 @@ returned by COMMAND is available via `executable-find'"
                                                (seq-every-p (lambda (el)
                                                               (stringp el))
                                                             l))))))
-  (list :connect (lambda (filter sentinel name environment-fn)
```

Comment:
If there's no real change in this, I think we should not touch this, it makes review code harder.

Reply:
Sorry, I thought it would be better to have everything be below < 80 columns.